catalog/app/containers/Profile/index.js

```diff
@@ -83,8 +84,8 @@ export class Profile extends React.PureComponent { // eslint-disable-line react/
 
   onDowngrade = () => {
     const { selectedPlan } = this.state;
-    const { dispatch } = this.props;
-    dispatch(updatePlan(selectedPlan));
+    const { updatePlan } = this.props;
+    updatePlan(selectedPlan);
```

Comment:
interesting. how does this work without `dispatch`?

Reply:
https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options

> If an object is passed, each function inside it is assumed to be a Redux action creator. An object with the same function names, but with every action creator wrapped into a dispatch call so they may be invoked directly, will be merged into the component’s props.

Reply:
btw since i don't use redux stuff for the pagination anymore, should i revert the changes related to `mapDispatchToProps`?

Reply:
yes